packages/core/src/state-management/repository.ts

```diff
@@ -738,6 +743,10 @@ export class Repository {
             this.logger.imp(
               `Successfully applied anchor commit ${anchorCommitCID} for stream ${state$.id}`
             )
+          } else {
+            this.logger.verbose(
+              `Successfully applied anchor commit ${anchorCommitCID} for stream ${state$.id}`
```

Comment:
this doesn't make sense why log the same thing in both cases?  if there are two cases we should have two different log things?